debs/SPECS/3.9.5/wazuh-agent/debian/rules

```diff
@@ -23,6 +23,20 @@ export INSTALLATION_DIR="/var/ossec"
 export INSTALLATION_SCRIPTS_DIR="${INSTALLATION_DIR}/packages_files/agent_installation_scripts"
 export JOBS="5"
 export DEBUG_ENABLED="no"
+export OS=$(lsb_release -si)
```

Comment:
These changes aren't necessary. In addition, you are changing the rules file of v3.9.5 and we won't release a new package of this version.